main.js

```diff
@@ -1,26 +1,28 @@
-const codeGenerator = require('./code-generator')
+const openAPI = require('./src/openapi');
+const constant = require('./src/constant');
 /**
  * @function _handleGenerate
  * @description OpenAPI generation when OpenAPI Initialization  
- * @param {string} base
+ * @param {UMLPackage} umlPackage
  * @param {string} path
  * @param {Object} options
  */
-function _handleGenerate(base, path, options) {
+function _handleGenerate(umlPackage, path, options) {
      // If options is not passed, get from preference
+
      options = options || getGenOptions();
-     // If base is not assigned, popup ElementPicker
-     if (!base) {
+     // If umlPackage is not assigned, popup ElementPicker
+     if (!umlPackage) {
           app.elementPickerDialog
                .showDialog("Select the package or project to generate from", null, null) //type.UMLPackage
-               .then(function({
+               .then(function ({
                     buttonId,
                     returnValue
                }) {
                     if (buttonId === "ok") {
```

Comment:
I think it's better to define an enum for buttonId, I see it's being used at lot of places.